baseapp/abci.go

```diff
@@ -356,6 +360,7 @@ func (app *BaseApp) handleQueryGRPC(handler GRPCQueryHandler, req abci.RequestQu
 
 	res, err := handler(ctx, req)
 	if err != nil {
+		err = grpcErrorToSDKError(err)
 		res = sdkerrors.QueryResult(err)
```

Comment:
nit: no need to re-assign, can just do:

```suggestion
		res = sdkerrors.QueryResult(grpcErrorToSDKError(err))
```